goodkey/good_key_test.go

```diff
@@ -262,7 +262,7 @@ func TestNonRefKey(t *testing.T) {
 func TestDBBlacklist(t *testing.T) {
 	exists := false
 	testCheck := func(context.Context, *sapb.KeyBlockedRequest) (*sapb.Exists, error) {
-		return &sapb.Exists{Exists: &exists}, nil
+		return &sapb.Exists{Exists: exists}, nil
```

Comment:
optional inline. I think the test would be clearer if line 275 completely overrode `testCheck`, rather than just overriding the closed-over `exists`, but that's just me.